### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Zapp/app](https://hosted.weblate.org/projects/zapp/app/).

It also includes following components:

* [Zapp/fastlane](https://hosted.weblate.org/projects/zapp/fastlane/)

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by eUgEntOptIc44** [comment]
Weblate has unfortunately **blocked** further contributions to the translations of zapp for the time being. As of writing this PR is waiting for a review for a few weeks and weblate's internal timeout for a possible merge has blocked the project.

File: fastlane/metadata/android/en/changelogs/52.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+* Fixed downloads not working
```
Comment (cemrich) [inline fastlane/metadata/android/en/changelogs/52.txt:1]: F-Droid falls back to en-US for fastlane metadata. So any translation tool needs to support this. "en" is most likely not going to work in this place.

File: app/src/main/res/values-en/strings.xml [hunk 2]
```diff
@@ -1,147 +1,121 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (cemrich) [inline app/src/main/res/values-en/strings.xml:1]: I am sorry I created conflicts in this file during refactoring / development :(

**Review by cemrich — CHANGES_REQUESTED** [review]

**Comment by cemrich** [comment]
Sorry, for my very late response! 

The work of the translators is amazing! But I need to have a very solid setup, so nothing breaks during development and - most importantly - release. My main concern right now is the fastlane `en` directory. In order for it to work as a F-Droid backup it [need to be](https://f-droid.org/docs/All_About_Descriptions_Graphics_and_Screenshots/#fastlane-struktur) an `en-US` directory. Can this be achieved with weblate?

I'll continue reviewing all weblate changes and try to put them all on the [feature/weblate](https://github.com/mediathekview/zapp/tree/feature/weblate) branch.

**Comment by cemrich** [comment]
I am verry, verry sorry it took me so long to look into this. I repaired the existing transifex integration which now can be used once again.

I your are still interested, you are free to join the translation team at https://explore.transifex.com/none-581/zapp-android-app/ I created a fresh French translation there.

**Pull request closed without merging**